pkg/cmd/pulumi/plugin_ls.go

```diff
@@ -90,7 +90,7 @@ type pluginInfoJSON struct {
 	Name         string  `json:"name"`
 	Kind         string  `json:"kind"`
 	Version      string  `json:"version"`
-	Size         int     `json:"size"`
+	Size         uint64  `json:"size"`
```

Comment:
I think this is OK, this gets turned into a float when serialising, so the lowest bits could potentially be rounded, but in practice we probably not going to ship plugins that are that large!